src/db.py

```diff
@@ -1,20 +1,56 @@
-from sqlalchemy import create_engine
-from sqlalchemy.orm import scoped_session, sessionmaker
-from sqlalchemy.ext.declarative import declarative_base
-from sqlalchemy.ext.declarative import declared_attr
+from datetime import datetime
+import requests
 
+from collegetown import collegetown_search
+from constants import CORNELL_DINING_URL
+from database import Base, Engine, Session
+from eatery_db import (
+    parse_campus_eateries,
+    parse_campus_hours,
+    parse_collegetown_eateries,
+    parse_menu_categories,
+    parse_menu_items,
+)
 
-engine = create_engine("sqlite:///data.sqlite3", convert_unicode=True)
-db_session = scoped_session(sessionmaker(autocommit=False, autoflush=False, bind=engine))
 
+def start_update():
+    try:
+        Base.metadata.drop_all(bind=Engine)
```

Comment:
rather than dropping all of the tables, maybe we could just drop the tables that contain the menu and hours? since the other data should remain the same

Reply:
If we do that, we won't know when cornell dining changes some general information about the eateries - e.g. changing payment methods (idk if this will be a huge issue though)

Reply:
I don't think the basic eatery information is refreshed that often, so I agree that it shouldn't be refreshed in our database every day.